packages/adblocker/test/matching.test.ts

```diff
@@ -11,19 +11,19 @@ import 'mocha';
 
 import { getDomain } from 'tldts-experimental';
 
-import CosmeticFilter from '../src/filters/cosmetic';
-import NetworkFilter, { isAnchoredByHostname } from '../src/filters/network';
+import CosmeticFilter from '../src/filters/cosmetic.js';
```

Comment:
I don't understand why all of these have `.js` extensions although we are importing TypeScript files. That's confusing to me.

Reply:
From what I understand this is exactly as modern typescript expects it to be when running in `nodenext` mode https://www.typescriptlang.org/docs/handbook/modules/reference.html#node16-nodenext